src/api/@types/fastify/index.d.ts

```diff
@@ -0,0 +1,14 @@
+import fastify from 'fastify';
+import { PgStore } from '../../../pg/pg-store';
+
+declare module 'fastify' {
+  export interface FastifyInstance<
+    HttpServer = Server,
+    HttpRequest = IncomingMessage,
+    HttpResponse = ServerResponse,
+    Logger = FastifyLoggerInstance,
+    TypeProvider = FastifyTypeProviderDefault,
+  > {
+    db: PgStore;
```

Comment:
Oof, this is the first major typing mistake/limitation I've seen in Fastify. Looks like this is the only work-around for decorators.

Reply:
Agreed, it's ugly AF 😬